lib/vsc/utils/optcomplete.py

```diff
@@ -91,7 +91,7 @@
 #      An array variable from which Bash reads the possible completions
```

Comment:
you can remove this module

Reply:
After our discussion this morning, keep it after all, or not?

Reply:
remove it

Reply:
It's imported from `generaloption`, not so trivial to flesh it out.

If you want it gone, please open an issue, won't do it in this PR.